pdns/dnsdistdist/docs/rules-actions.rst

```diff
@@ -838,6 +838,14 @@ The following actions exist.
 
   Let these packets go through.
 
+.. function::ClearRecordTypesResponseAction(types)
+
+  .. versionadded:: 1.8.0
+
+  Removes given type(s) records from the response.
```

Comment:
some extra text would be good that the operator has to be careful about accidentally turning an answer into a NODATA response without a SOA record in the additional section. This could be prevented by using a `NotRule{QTypeRule(XXX)}` where XXX are the record types being stripped in the response.

Reply:
Alright, thanks! I added a word about that and an example code.